### ISSUE TITLE
Cannot reproduce the result in Paper using the data of Nonlocal-Net

### ISSUE THREAD

**Issue by justimyhxu** [facebookresearch/SlowFast#54]
We have used the official code and run the config SLOWONLY_8x8_R50.yaml. The setting is the same as the setting stated in the paper. And we get the Kinetics data in Nonlocal-Net. But our reproduced result of 8*8 in val is 73.97, nearly 1% drop compared in your model zoo. 
I want to know the reason and what details do we ignore？

**Comment by takatosp1** [comment]
Hi,
Thanks for using the codebase! It would be useful if you could share me some detail from your place, then I can try to help you to see what is missing from your side.
I am not sure what is the source of your train/ Val set. Could you please share me what data you were using (maybe a link?) and how many training samples do you have? Also preprocessing matters a lot, could you share me what format and what resolution you use?  Beside of that, could you also share what pyav and ffmpeg version？
Thanks

**Comment by justimyhxu** [comment]
We follow this link(https://github.com/facebookresearch/video-nonlocal-net/blob/master/DATASET.md) to get the Kinetics data. And we use the video loader to seek frame from the raw video with the resolution of 256. Our pyav version is 6.2.0 and ffmpeg version is 4.1.3.

**Comment by takatosp1** [comment]
Hi,
Thanks for sharing the details!
For the resolution, please remember to rescale the short side to 256.
The data you were using is ~5% smaller than the data used to train the model, which might lead in ~0.5% difference in performance. The general variance on Kinetics dataset is ~0.3%.

**Comment by gurkirt** [comment]
A copy of Xia's dataset can be found on https://www.dropbox.com/s/wcs01mlqdgtq4gn/compress.tar.gz?dl=1.

@justimyhxu, did you get the data from Xia via xiaolonw@cs.cmu.edu or downloaded yourself from youtube or from above link?

**Comment by BoPang1996** [comment]
I got similar results with @justimyhxu when adopting data shared by Xiaolong Wang in Nonlocal repo. 

I use the config configs/Kinetics/SLOWFAST_8x8_R50.yaml. The date I use is shared by Xiaolong Wang in Nonlocal which contains 234643 training videos and 19761 val videos. I do not have 16 nodes, so I trained the model on 2x8 V100 cards with 8 mini-batch on each card. The base learning rate is scaled to 0.2. The top-1 accuracy is 75.8, 1.2 lower than the official results.

For SLOWFAST_8x8_R101_101_101.yaml, the reproduced top-1 acc is 77.2, 0.7 lower than the official results.

For SLOW_8x8_R50.yaml, the reproduced top-1 acc is 74.0, 0.8 lower than the official results. The same with @justimyhxu.

@takatosp1 Would you please mention this in model_zoo for the convenience of other users and researchers? After all the intact Kinetics dataset is not accessible, this will create a more convenient and fair scientific research environment for the video community.
